### DISCUSSION TITLE
How to scale with Nearest Neighbour

### DISCUSSION THREAD

**Discussion by SebastianRiesel** [AlmasB/FXGL#1137] [Q&A]
Hello!
Today I was trying to scale some textures,
```
texture.setScaleX(scale);
texture.setScaleY(scale);
```
 but when I did so, my pixel art became all blurry. Is there any way I can use nearest neigbour scaling in fxgl? 
![grafik](https://user-images.githubusercontent.com/68468913/147825436-c23c6447-00bb-404e-a532-35fd416622a5.png)

**Comment by AlmasB** [comment]
Hi, you can try using:

```
ImagesKt.resize(texture.getImage(), width, height);
```

which will resize the image, rather than scale the resulting view -- it might end up being a bit better.
